components/mpas-ocean/bld/namelist_files/namelist_definition_mpaso.xml

```diff
@@ -581,6 +581,54 @@ Valid values: values between zero and bottom depth
 Default: Defined in namelist_defaults.xml
 </entry>
 
+<entry id="config_GM_Griffies_min_depth" type="real"
+	category="GM_eddy_parameterization" group="GM_eddy_parameterization">
+minimum depth to start integral for the Griffies parameter
+
+Valid values: positive reals
+Default: Defined in namelist_defaults.xml
+</entry>
+
+<entry id="config_GM_Griffies_max_depth" type="real"
+	category="GM_eddy_parameterization" group="GM_eddy_parameterization">
+maximum depth for the integral of the Griffies parameter
+
+Valid values: positive reals greater than min_depth
+Default: Defined in namelist_defaults.xml
+</entry>
+
+<entry id="config_GM_Griffies_compute_levels" type="char*1024"
+	category="GM_eddy_parameterization" group="GM_eddy_parameterization">
+Compute the min and max vertical k indices based on config_GM_Griffies_min_depth, config_GM_Griffies_max_depth just one on init (for 'init')  or every time step ('every_timestep'). For z-level and z-star vertical coordinates it is better to compute on init for better performance, to avoid re-computing the min and max level for every edge and every timestep.
```

Comment:
@vanroekel  I added this option because on all our standard runs, we don't need to recompute the min/max level on every edge, for every time step.